velox/exec/TableWriter.cpp

```diff
@@ -255,7 +255,7 @@ void TableWriter::updateStats(const connector::DataSink::Stats& stats) {
         "numWrittenFiles", RuntimeCounter(stats.numWrittenFiles));
   }
   if (!stats.spillStats.empty()) {
```

Comment:
You might need to pass spillStats_ from table writer to HiveDataSink::Stats? Thanks!